# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libmspub/hotfix.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libmspub-0.1.4/src/lib/MSPUBMetaData.h.vanilla	2024-05-22 12:31:42.154000000 +0200
+++ libmspub-0.1.4/src/lib/MSPUBMetaData.h	2024-05-22 12:31:51.973000000 +0200
@@ -10,6 +10,8 @@
 #ifndef INCLUDED_MSPUBMETADATA_H
 #define INCLUDED_MSPUBMETADATA_H
 
+#include <inttypes.h>
+
 #include <map>
 #include <utility>
 #include <vector>
